src/encoded/static/scss/encoded/modules/_status.scss

```diff
@@ -0,0 +1,237 @@
+// All status indicator colors. As statuses change in the schemas, should see if corresponding
+// changes are needed here.
+$status-colors: (
```

Comment:
And now we have to define colors for each status text string?   I guess it's not trivial to set this in SASS programattically.

Reply:
Yes, which is what we were doing before this change as well, though in a very inconsistent way (sometimes in a loop, sometimes statically). Now all statuses are mapped to corresponding colors in a loop.